jujugui/static/gui/src/app/components/status/test-status.js

```diff
@@ -585,4 +645,74 @@ describe('Status', function() {
     expect(comp.output).toEqualJSX(expectedOutput);
   });
 
+  it('can navigate to applications from the app list', () => {
+    const comp = render(makeDB());
+    comp.output.props.children.props.children[2].props.rows[0].columns[0]
```

Comment:
Please decompose this into multiple assignments, so that it is easier to read.
Like 
```javascript
const table = ...;
const row = ...;
// etc
```
Please do the same in the tests below.